Dockerfile

```diff
@@ -3,9 +3,11 @@ FROM ubuntu:20.04
 WORKDIR /root/
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && \
-    apt-get install --no-install-recommends -y curl erlang gawk gnupg2 groovy libicu-dev liblttng-ust0 nodejs npm openjdk-11-jre-headless php python ruby vim wget
+    apt-get install --no-install-recommends -y curl erlang gawk gnupg2 groovy libicu-dev liblttng-ust0 nodejs npm openjdk-11-jre-headless php python ruby snapd vim wget
 ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
 
+RUN snap install julia --classic
```

Comment:
This change causes the Docker image build to fail: https://travis-ci.org/github/mkasberg/script-seed/builds/734405155

Snaps don't generally work inside a Docker container. Fortunately, `julia` is available from `apt-get`, so you can simply add `julia` to the `apt-get` line above instead of installing it as a Snap package.

Reply:
Thank you. I was not knowing they have made it available in the apt-get. I have changed the dockerfile, it seems to be working fine now as I see that all the results have passed.